src/main/java/au/org/emii/ncdfgenerator/AttrWrapper.java

```diff
@@ -0,0 +1,33 @@
+
+package au.org.emii.ncdfgenerator;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import org.w3c.dom.Node;
+import org.w3c.dom.NamedNodeMap;
+
+
+class AttrWrapper implements Iterable<Node> {
```

Comment:
hmm, I wonder if this is git screwing things up... this wrapper class was already merged in #41